src/components/document-metadata-modal.js

```diff
@@ -370,7 +370,9 @@ function DocumentMetadataModal({
         {canCreateSequence && (
           <FormItem>
             <Checkbox checked={generateSequence} onChange={handleGenerateSequenceChange}>
-              <Info tooltip={t('sequenceInfo')} iconAfterContent>{t('generateSequence')}</Info>
+              <Info tooltip={t('sequenceInfo')} iconAfterContent>
+                <span className="u-label">{t('generateSequence')}</span>
```

Comment:
Fixing a UI inconsistency here, making this label have the same styling (color) as the others.

Reply:
:astonished: